CHANGELOG.md

```diff
@@ -28,6 +28,7 @@ The following is a curated list of changes in the Enact sandstone module, newest
 - `sandstone/FormCheckboxItem` to show correct color for `slotBefore` icon in disabled state when focused
 - `sandstone/ImageItem` to resize the image properly
 - `sandstone/Input` button label when default value is `0`
+- `sandstone/InputField` cursor not to jump unexpectedly when mouse down
```

Comment:
Please rebase your branch with the latest develop and put the log in the right place.

Reply:
OK. it's rebased and fixed.